aiohttp/file_sender.py

```diff
@@ -152,16 +153,48 @@ def send(self, request, filepath):
         if not ct:
             ct = 'application/octet-stream'
 
-        resp = self._response_factory()
+        file_size = st.st_size
+
+        status = 200
+        start = 0
+        end = file_size - 1
+
+        # Handle 206 range response if requested
+        if 'range' in request.headers:
```

Comment:
i think it make sense to move range related functionality to helper module or even to separate module. and create `Range` class, it could be useful for external libraries and applications